File: Python/6kyu/MultiplesOf3Or5.py
def solution(number):
    return sum(n if(n % 3 == 0 or n % 5 == 0) else 0 for n in range(0, number))


def solution_2(number):
    return sum(x for x in range(number) if x % 3 == 0 or x % 5 == 0)


result = solution_2(16)
print(result)

File: JavaScript/6kyu/MultiplesOf3Or5.js
function solution(number) {
  var sum = 0;

  for (let i = 3; i < number; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      sum += i;
    }
  }

  return sum;
}

var result = solution(10);
console.log(result);
